R/spell-check.R

```diff
@@ -1,79 +1,7 @@
-#' Spell checking
-#'
-#' Runs a spell check on text fields in the package description file and
-#' manual pages. Hunspell includes dictionaries for \code{en_US} and \code{en_GB}
-#' by default. Other languages require installation of a custom dictionary, see
-#' the \href{https://cran.r-project.org/package=hunspell/vignettes/intro.html#system_dictionaries}{hunspell vignette}
-#' for details.
-#'
+#' @rdname devtools-deprecated
 #' @export
-#' @rdname spell_check
-#' @param pkg package description, can be path or package name.  See
-#'   \code{\link{as.package}} for more information
-#' @param ignore character vector with words to ignore. See
-#'   \code{\link[hunspell:hunspell]{hunspell}} for more information
-#' @param dict a dictionary object or language string. See
-#'   \code{\link[hunspell:hunspell]{hunspell}} for more information
-spell_check <- function(pkg = ".", ignore = character(), dict = "en_US"){
-
-  check_suggested("hunspell")
-
+spell_check <- function(pkg = ".", vignettes = TRUE, lang = "en_GB"){
+  .Deprecated("spelling::spell_check_package()", package = "devtools")
```

Comment:
Is there a good reason to deprecate this, rather than soft-deprecating it?

Reply:
No, I just followed the example from the `use_this` functions. We can also keep it there as a wrapper if you'd like.

Reply:
We deprecated the usethis functions because we're going to attach usethis so generally you'll get the new versions (the deprecation is just for people importing via namespace)

Reply:
OK i have updated the PR to not deprecate.